### PR TITLE
Update to robolectric 3.1.2

### PR DESCRIPTION
**Pull request by jaredsburrows**
@JakeWharton Update to Robolectric 3.1.2

### CODE REVIEW

File: butterknife/src/test/java/butterknife/internal/UtilsTest.java [hunk 1]
```diff
@@ -14,7 +15,7 @@
 import static org.junit.Assert.fail;
 
 @RunWith(RobolectricTestRunner.class)
-@Config(manifest = Config.NONE)
+@Config(constants = BuildConfig.class)
```
Comment (JakeWharton) [inline butterknife/src/test/java/butterknife/internal/UtilsTest.java]: Why do we need this and the resource file?

File: butterknife-integration-test/build.gradle [hunk 2]
```diff
@@ -42,4 +42,5 @@ dependencies {
   testCompile deps.junit
   testCompile deps.truth
   testCompile deps.robolectric
+  testCompile deps.openglApi
```
Comment (JakeWharton) [inline butterknife-integration-test/build.gradle]: Why?

Comment (jaredsburrows) [inline butterknife-integration-test/build.gradle]: A bug when using API 23: https://github.com/robolectric/robolectric/issues/1932.

File: butterknife/src/test/java/butterknife/internal/UtilsTest.java (hunk 1 above)
Comment (jaredsburrows) [inline butterknife/src/test/java/butterknife/internal/UtilsTest.java]: @JakeWharton Reverted.

**Pull request merged**